addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -118,7 +118,7 @@
                                 </div>
                                 <div class="row mx-n2 mt8">
                                     <t t-foreach="channels_my[:3]" t-as="channel">
-                                        <div class="col-md-4 col-sm-6 px-2 col-xs-12 d-flex flex-grow-1">
```

Comment:
I just check and it seems that this line has been modified for the last time 3 years ago.

Do we know why this only appears in v16? Is it tied to a bootstrap lib update?

Reply:
Hi :) From what I understand, it is tied to a bootstrap lib update. On bootstrap 4 col-xx-x has max-width attribute and because of that it wasn't a problem before. Now bootstrap 5 uses normal width attribute and flex basically just ignores the attribute.

Reply:
Do you mind explaining that into the commit message?

Then we can proceed, thanks!

Reply:
Done! :smile: